packages/extensions/core/src/lib/pipeline/plugins/schema-validation/swagger-schema-validator.ts

```diff
@@ -0,0 +1,22 @@
+import addFormats from "ajv-formats";
+import { JsonSchemaValidator } from "./json-schema-validator";
+
+export class SwaggerSchemaValidator extends JsonSchemaValidator {
+  public constructor() {
+    super();
+    addFormats(this.ajv);
+
+    this.ajv.addSchema(
+      require("C:/dev/azsdk/autorest/packages/libs/autorest-schemas/swagger.json"),
```

Comment:
Does this path (and the next one) need to be updated?

Reply:
haha completly missed that copied over from a sandbox script.

Reply:
No worries!